src/asciidoc/docs/develop/testingTemplates.adoc

```diff
@@ -0,0 +1,171 @@
+= Testing templates
+:page-slug: testing-templates
+:page-description: Guide to testing Koji templates.
+
+There are many powerful testing tools built into the Koji platform.
+Typically, you will begin testing in the Koji editor.
+Later, you'll want to publish the template as *Unlisted*, so you can test it in a production environment before giving access to general users.
+Some of the most common testing methods are listed below.
+
+== Testing in the Koji editor
+
+The Koji editor provides various ways to preview a template's behavior.
+
+=== Previewing the Koji
+
+The *Live preview* tab allows you to toggle between *Preview* mode, which shows what the Koji looks like when running, and *Editing* mode, which allows you to perform remixing in the preview.
```

Comment:
These sections still feel a bit disjointed, especially the separation of the different contexts from the other live preview functionality. I think we need to bring the various contexts together to explain what they are. Then, maybe there's a different section for the "remote" options. Something like, "Previewing the Koji on different devices." That might not be the right title, but I think explaining what makes the options "remote" is the idea.